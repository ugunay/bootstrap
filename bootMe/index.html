<!doctype html>
<html lang="en">

<head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <!-- Bootstrap CSS -->
    <link rel="stylesheet" type="text/css" href="css/fontawesome-all.min.css">
    <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
    <link rel="stylesheet" type="text/css" href="css/lightbox.min.css">
    <link rel="stylesheet" type="text/css" href="css/main.css">
    <!-- <link rel="icon" 
      type="images/logo.png" 
      href="http://example.com/myicon.png"> -->
    <!-- <link href="https://fonts.googleapis.com/css?family=Fredoka+One" rel="stylesheet"> -->
    <!-- <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet"> -->
    <title>Boot Me Title</title>
</head>

<body>
    <nav class="navbar navbar-expand-md navbar-dark bg-dark fixed-top">
        <div class="container">
            <button class="navbar-toggler" data-toggle="collapse" data-target="#nvbCollapse">
                <span class="navbar-toggler-icon"></span>
            </button>
            <a href="#" class="navbar-brand">BootMe</a>
            <div id="nvbCollapse" class="collapse navbar-collapse">
                <ul class="navbar-nav ml-auto">
                    <li class="nav-item">
                        <a href="#headerSection" class="nav-link">Home</a>
                    </li>
                    <li class="nav-item">
                        <a href="#features" class="nav-link">About Us</a>
                    </li>
                    <li class="nav-item">
                        <a href="#caseStudy" class="nav-link">Our Works</a>
                    </li>
                    <li class="nav-item">
                        <a href="#contact" class="nav-link">Contact</a>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <header id="headerSection">
        <div class="overlay">
            <div class="container">
                <div class="row text-center">
                    <div class="col">
                        <h1 class="display-4"> <span>BootMe</span> Landing Theme</h1>
                        <p class="text-muted"><em>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer sed
                                elit,
                                a scelerisque urna.</em></p>
                        <button type="button" class="btn btn-outline-primary">Action</button>
                    </div>
                </div>
            </div>
        </div>
    </header>

    <section id="features" class="pt-5">
        <div class="container">
            <div class="row mt-5">
                <div class="col-md-3 text-center">
                    <div class="feature">
                        <div class="feature-top">
                            <i class="fab fa-ravelry"></i>
                            <h3 class="mt-3">SLEEK DESIGN</h3>
                        </div>

                        <p>Nullam lacus metus, tincidunt a blandit sed, porta in ipsum. Donec in mattis libero.</p>
                    </div>
                </div>
                <div class="col-md-3 text-center">
                    <div class="feature">
                        <div class="feature-top">
                            <i class="fas fa-cog"></i>
                            <h3 class="mt-3">CLEAN CODE</h3>
                        </div>

                        <p>Nullam lacus metus, tincidunt a blandit sed, porta in ipsum. Donec in mattis libero.</p>
                    </div>
                </div>
                <div class="col-md-3 text-center">
                    <div class="feature">
                        <div class="feature-top">
                            <i class="fas fa-leaf"></i>
                            <h3 class="mt-3">CREATIVE IDEAS</h3>
                        </div>

                        <p>Nullam lacus metus, tincidunt a blandit sed, porta in ipsum. Donec in mattis libero.</p>
                    </div>
                </div>
                <div class="col-md-3 text-center">
                    <div class="feature">
                        <div class="feature-top">
                            <i class="fas fa-podcast"></i>
                            <h3 class="mt-3">FREE SUPPORT</h3>
                        </div>

                        <p>Nullam lacus metus, tincidunt a blandit sed, porta in ipsum. Donec in mattis libero.</p>
                    </div>
                </div>
            </div>
        </div>
    </section>

    <section id="ourHistory">
        <div class="container">
            <div class="row mt-5">
                <div class="col-lg-6">
                    <img src="images/appleMonitor.png" class="img-fluid" alt="Our History">
                </div>
                <div class="col-lg-6">
                    <h2 class="mb-5">OUR HISTORY</h2>
                    <p>Nunc fermentum, nulla sed eleifend sollicitudin, eu volutpat neque ligula ac magna. Morbi congue
                        rutrum sem quis mollis. Suspendisse varius consectetur lobortis. Aliquam in fringilla massa. Fusce viverra nisi quis ligula sodales rhoncus. Vivamus eu dolor ac lacus interdum ultricies in at metus.
                        Pellentesque interdum consectetur ipsum, vitae quis.</p>
                    <p>Nunc fermentum, nulla sed eleifend sollicitudin, eu volutpat neque ligula ac magna. Morbi congue
                        rutrum sem quis mollis. Suspendisse varius consectetur lobortis. Aliquam in fringilla massa. Fusce viverra nisi quis ligula sodales rhoncus. Vivamus eu dolor ac lacus interdum ultricies in at metus.
                        Pellentesque interdum consectetur ipsum, vitae quis.</p>
                    <button type="button" class="btn btn-outline-primary">Check Our Work</button>
                </div>
            </div>
        </div>
    </section>

    <section id="team" class="mt-5">
        <div class="container">
            <div class="row">
                <div class="col-md-3">
                    <div class="card text-center">
                        <div class="card-block">
                            <img src="images/person1.jpg" alt="" class="img-fluid rounded-circle my-4">
                            <h3>John Doe</h3>
                            <h5 class="text-muted">CEO</h5>
                            <p>Vestibulum ut purus ultrices, bibendum ligula id, suscipit sem.</p>
                            <div class="d-flex flex-row justify-content-center">
                                <div class="p-4">
                                    <a href="#"><i class="fab fa-facebook"></i></a>
                                </div>
                                <div class="p-4">
                                    <a href="#"><i class="fab fa-twitter"></i></a>
                                </div>
                                <div class="p-4">
                                    <a href="#"><i class="fab fa-linkedin"></i></a>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <div class="col-md-3">
                    <div class="card text-center">
                        <div class="card-block">
                            <img src="images/person2.jpg" alt="" class="img-fluid rounded-circle my-4">
                            <h3>Julia Doe</h3>
                            <h5 class="text-muted">CTO</h5>
                            <p>Vestibulum ut purus ultrices, bibendum ligula id, suscipit sem.</p>
                            <div class="d-flex flex-row justify-content-center">
                                <div class="p-4">
                                    <a href="#"><i class="fab fa-facebook"></i></a>
                                </div>
                                <div class="p-4">
                                    <a href="#"><i class="fab fa-twitter"></i></a>
                                </div>
                                <div class="p-4">
                                    <a href="#"><i class="fab fa-linkedin"></i></a>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <div class="col-md-3">
                    <div class="card text-center">
                        <div class="card-block">
                            <img src="images/person3.jpg" alt="" class="img-fluid rounded-circle my-4">
                            <h3>Jack Doe</h3>
                            <h5 class="text-muted">Lead Coder</h5>
                            <p>Vestibulum ut purus ultrices, bibendum ligula id, suscipit sem.</p>
                            <div class="d-flex flex-row justify-content-center">
                                <div class="p-4">
                                    <a href="#"><i class="fab fa-facebook"></i></a>
                                </div>
                                <div class="p-4">
                                    <a href="#"><i class="fab fa-twitter"></i></a>
                                </div>
                                <div class="p-4">
                                    <a href="#"><i class="fab fa-linkedin"></i></a>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <div class="col-md-3">
                    <div class="card text-center">
                        <div class="card-block">
                            <img src="images/person4.jpg" alt="" class="img-fluid rounded-circle my-4">
                            <h3>Mary Doe</h3>
                            <h5 class="text-muted">Lead Designer</h5>
                            <p>Vestibulum ut purus ultrices, bibendum ligula id, suscipit sem.</p>
                            <div class="d-flex flex-row justify-content-center">
                                <div class="p-4">
                                    <a href="#"><i class="fab fa-facebook"></i></a>
                                </div>
                                <div class="p-4">
                                    <a href="#"><i class="fab fa-twitter"></i></a>
                                </div>
                                <div class="p-4">
                                    <a href="#"><i class="fab fa-linkedin"></i></a>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

    <section id="whatWeDo" class="py-5">
        <div class="contanier">
            <div class="row">
                <div class="col-md-12 text-center">
                    <h2>What Are We Doing?</h2>
                    <p class="text-muted"><em>Nunc ante metus, euismod nec purus at, iaculis ornare felis</em></p>
                </div>
                <div class="col-md-6 text-center mx-auto">
                    <a href="#" class="video" data-video="https://www.youtube.com/embed/k5E2AVpwsko" data-toggle="modal"
                        data-target="#videoModal">
                        <i class="fas fa-play-circle fa-5x"></i>
                    </a>
                </div>
            </div>
        </div>
    </section>

    <div id="videoModal" class="modal fade">
        <div class="modal-dialog modal-lg">
            <div class="modal-content">
                <div class="modal-body">
                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                        <span aria-hidden="true">&times;</span>
                    </button>
                    <iframe id="iframeYoutube" width="100%" height="400" src="https://www.youtube.com/embed/k5E2AVpwsko"
                        frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
                        allowfullscreen></iframe>
                </div>
                <div class="modal-footer">
                    <button id="btnClose" type="button" class="btn btn-primary" data-dismiss="modal">Close</button>
                </div>
            </div>
        </div>
    </div>

    <section id="caseStudy" class="py-5">
        <div class="container">
            <div class="row">
                <div class="col-md-6">
                    <div class="case-area text-center">
                        <i class="fas fa-podcast"></i>
                        <h3>ACCUMULATIVE CREATIVE IDEAS</h3>
                        <p>Cras ut mauris sed enim tincidunt mattis. Vestibulum volutpat, mi vitae volutpat faucibus,
                            magna nisi sollicitudin ipsum, vel fringilla nisl eros ac nisl. Vestibulum ut purus
                            ultrices, bibendum ligula id, suscipit sem. Sed vehicula quam sit amet urna consequat
                            dignissim.</p>
                        <button type="button" class="btn btn-outline-primary">Action</button>
                    </div>
                </div>
                <div class="col-md-6">
                    <div class="case-image">
                        <img src="images/caseStudy.jpg" class="img-fluid">
                    </div>
                </div>
            </div>
        </div>
    </section>

    <section id="portfolio">
        <div class="container-fluid">
            <div class="row">
                <div class="col-md-3">
                    <a href="images/portfolio1.jpg" data-lightbox="image-1" data-title="Web Design">
                        <div class="image-container">
                            <img src="images/portfolio1.jpg" class="img-fluid">
                            <div class="image-overlay">
                                <div class="image-text">
                                    <i class="fas fa-search"></i>
                                    <h4>WEB DESIGN</h4>
                                    <p><em>code / design</em></p>
                                </div>
                            </div>
                        </div>
                    </a>
                </div>
                <div class="col-md-3">
                    <a href="images/portfolio2.jpg" data-lightbox="image-1" data-title="Web Design">
                        <div class="image-container">
                            <img src="images/portfolio2.jpg" class="img-fluid">
                            <div class="image-overlay">
                                <div class="image-text">
                                    <i class="fas fa-search"></i>
                                    <h4>WEB DESIGN</h4>
                                    <p><em>code / design</em></p>
                                </div>
                            </div>
                        </div>
                    </a>
                </div>
                <div class="col-md-3">
                    <a href="images/portfolio3.jpg" data-lightbox="image-1" data-title="Web Design">
                        <div class="image-container">
                            <img src="images/portfolio3.jpg" class="img-fluid">
                            <div class="image-overlay">
                                <div class="image-text">
                                    <i class="fas fa-search"></i>
                                    <h4>WEB DESIGN</h4>
                                    <p><em>code / design</em></p>
                                </div>
                            </div>
                        </div>
                    </a>
                </div>
                <div class="col-md-3">
                    <a href="images/portfolio4.jpg" data-lightbox="image-1" data-title="Web Design">
                        <div class="image-container">
                            <img src="images/portfolio4.jpg" class="img-fluid">
                            <div class="image-overlay">
                                <div class="image-text">
                                    <i class="fas fa-search"></i>
                                    <h4>WEB DESIGN</h4>
                                    <p><em>code / design</em></p>
                                </div>
                            </div>
                        </div>
                    </a>
                </div>


            </div>
            <div class="row">
                <div class="col-md-3">
                    <a href="images/portfolio5.jpg" data-lightbox="image-1" data-title="Web Design">
                        <div class="image-container">
                            <img src="images/portfolio5.jpg" class="img-fluid">
                            <div class="image-overlay">
                                <div class="image-text">
                                    <i class="fas fa-search"></i>
                                    <h4>WEB DESIGN</h4>
                                    <p><em>code / design</em></p>
                                </div>
                            </div>
                        </div>
                    </a>
                </div>
                <div class="col-md-3">
                    <a href="images/portfolio6.jpg" data-lightbox="image-1" data-title="Web Design">
                        <div class="image-container">
                            <img src="images/portfolio6.jpg" class="img-fluid">
                            <div class="image-overlay">
                                <div class="image-text">
                                    <i class="fas fa-search"></i>
                                    <h4>WEB DESIGN</h4>
                                    <p><em>code / design</em></p>
                                </div>
                            </div>
                        </div>
                    </a>
                </div>
                <div class="col-md-3">
                    <a href="images/portfolio7.jpg" data-lightbox="image-1" data-title="Web Design">
                        <div class="image-container">
                            <img src="images/portfolio7.jpg" class="img-fluid">
                            <div class="image-overlay">
                                <div class="image-text">
                                    <i class="fas fa-search"></i>
                                    <h4>WEB DESIGN</h4>
                                    <p><em>code / design</em></p>
                                </div>
                            </div>
                        </div>
                    </a>
                </div>
                <div class="col-md-3">
                    <a href="images/portfolio8.jpg" data-lightbox="image-1" data-title="Web Design">
                        <div class="image-container">
                            <img src="images/portfolio8.jpg" class="img-fluid">
                            <div class="image-overlay">
                                <div class="image-text">
                                    <i class="fas fa-search"></i>
                                    <h4>WEB DESIGN</h4>
                                    <p><em>code / design</em></p>
                                </div>
                            </div>
                        </div>
                    </a>
                </div>


            </div>
        </div>
    </section>

    <section id="testimonial" class="text-center py-5">
        <div class="container">
            <div class="row">
                <div class="col-md-10 mx-auto">
                    <div id="carouselTestimonial" class="carousel slide" data-ride="carousel">
                        <ol class="carousel-indicators">
                            <li data-target="#carouselTestimonial" data-slide-to="0" class="active"></li>
                            <li data-target="#carouselTestimonial" data-slide-to="1"></li>
                            <li data-target="#carouselTestimonial" data-slide-to="2"></li>
                        </ol>
                        <div class="carousel-inner ">
                            <div class="carousel-item active">
                                <img src="images/carousel1.jpg" class="d-block w-100" alt="...">
                            </div>
                            <div class="carousel-item">
                                <img src="images/carousel2.jpg" class="d-block w-100" alt="...">
                            </div>
                            <div class="carousel-item">
                                <img src="images/carousel3.jpg" class="d-block w-100" alt="...">
                            </div>
                        </div>
                        <a class="carousel-control-prev" href="#carouselTestimonial" role="button" data-slide="prev">
                            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
                            <span class="sr-only">Previous</span>
                        </a>
                        <a class="carousel-control-next" href="#carouselTestimonial" role="button" data-slide="next">
                            <span class="carousel-control-next-icon" aria-hidden="true"></span>
                            <span class="sr-only">Next</span>
                        </a>
                    </div>
                </div>
            </div>

        </div>
    </section>

    <section id="contact" class="py-5">
        <div class="container">
            <div class="row">
                <div class="col-md-6">
                    <h2>CONTACT US</h2>
                    <p>Mauris massa lacus, feugiat vel malesuada vel, tincidunt in massa.</p>
                    <h4>Address</h4>
                    <p>Canada</p>
                    <h4>Email</h4>
                    <p>info@bootme.com</p>
                    <h4>Phone</h4>
                    <p>111 222 3333</p>
                    <div class="d-flex flex-row">
                        <div class="p-2">
                            <a href="#">
                                <i class="fab fa-facebook fa-2x"></i>
                            </a>
                        </div>
                        <div class="p-2">
                            <a href="#">
                                <i class="fab fa-twitter fa-2x"></i>
                            </a>
                        </div>
                        <div class="p-2">
                            <a href="#">
                                <i class="fab fa-linkedin fa-2x"></i>
                            </a>
                        </div>
                    </div>
                </div>
                <div class="col-md-6">
                    <form>
                        <div class="form-group">
                            <div class="input-group input-group-lg">
                                <span class="input-group-addon my-auto">
                                    <i class="fas fa-user"></i>
                                </span>
                                <input type="text" class="form-control" placeholder="Name">
                            </div>
                        </div>
                        <div class="form-group">
                            <div class="input-group input-group-lg">
                                <span class="input-group-addon">
                                    <i class="fas fa-envelope my-1"></i>
                                </span>
                                <input type="email" class="form-control" placeholder="Email">
                            </div>
                        </div>
                        <div class="form-group ">
                            <div class="input-group input-group-lg">
                                <span class="input-group-addon">
                                    <i class="fas fa-pencil-alt my-5"></i>
                                </span>
                                <textarea class="form-control" placeholder="Message" rows="5"></textarea>
                            </div>
                        </div>
                        <input type="submit" value="SUBMIT" class="btn btn-primary btn-block">
                    </form>
                    </form>
                </div>
            </div>
        </div>
    </section>

    <footer id="main-footer" class="p-2">
        <div class="container-fluid">
            <div class="row text-center">
                <div class="col-m-6 mx-auto ">
                    <p class="lead">No Copyright &copy; 2019 BootMe</p>
                </div>
            </div>
        </div>
    </footer>
    

    <script src="js/jquery-3-3-1.min.js"></script>
    <script src="js/popper.min.js"></script>
    <script src="js/bootstrap.bundle.min.js"></script>
    <script src="js/lightbox.min.js"></script>
    <script src="js/smooth-scroll.min.js"></script>

    <!-- <script>
        new WOW().init();
    </script> -->
    <script src="js/main.js"></script>
    <script>
            var scroll = new SmoothScroll('a[href*="#"]');
    </script>
</body>

</html>