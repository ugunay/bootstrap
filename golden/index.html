<!doctype html>
<html lang="en">

<head>
  <!-- Required meta tags -->
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <!-- Bootstrap CSS -->
  <link rel="stylesheet" type="text/css" href="css/fontawesome-all.min.css">
  <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
  <link rel="stylesheet" type="text/css" href="css/animate.css">
  <link rel="stylesheet" type="text/css" href="css/main.css">
  <link rel="icon" 
      type="images/logo.png" 
      href="http://example.com/myicon.png">
  <link href="https://fonts.googleapis.com/css?family=Fredoka+One" rel="stylesheet">
  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
  <title>Bootstrap Test Title</title>
</head>

<body>
  <section id="cover">
    <div class="container">
      <div class="row">
        <nav class="navbar navbar-expand-lg navbar-light fixed-top">
          <a class="navbar-brand" href="#"><img id="logoImage" src="images/logo.png" class="d-inline-block align-top "
              alt="Golden Website"></a>
          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav"
            aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
          </button>

          <div class="collapse navbar-collapse justify-content-around" id="navbarNav">
            <ul class="navbar-nav">
              <li class="nav-item"><a href="#cover" class="nav-link text-uppercase">Home</a></li>
              <li class="nav-item"><a href="#service" class="nav-link text-uppercase">Services</a></li>
              <li class="nav-item"><a href="#portfolio" class="nav-link text-uppercase">Portfolio</a></li>
              <li class="nav-item"><a href="#aboutus" class="nav-link text-uppercase">About</a></li>
              <li class="nav-item"><a href="#contact" class="nav-link text-uppercase">CONTACT</a></li>
            </ul>
          </div>
        </nav>
        <div class="row text-center">
          <div class="col-md-12">
            <h1>Welcome to Our Sports Center</h1>
          </div>
          <div class="col-md-12">
            <h2>NICE TO MEET YOU</h2>
          </div>
          <div class="col-md-12">
            <a href="#"><button type="button" class="btn btn-primary btn-lg">CLICK FOR MORE</button> </a>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section id="service">
    <div class="container">
      <div class="service-top text-center wow fadeInUp" data-wow-duration="2s">
        <h3>SERVICES</h3>
        <p>Here are our services</p>
      </div>
      <div class="service-content text-center">
        <div class="row">
          <div class="col-md-4 wow fadeInLeft" data-wow-duration="2s">
            <div class="item-circle">
              <i class="fas fa-swimmer fa-4x"></i>
            </div>
            <h4>Swimming</h4>
            <p>Maecenas bibendum lorem vitae turpis ullamcorper laoreet. Proin semper dictum risus in porta. In hac
              habitasse platea dictumst. Duis tincidunt, purus tortor fringilla orci, sit amet maximus urna velit eu
              urna.</p>
          </div>
          <div class="col-md-4 wow fadeInUp" data-wow-duration="2s">
            <div class="item-circle">
              <i class="fas fa-basketball-ball fa-4x"></i>
            </div>
            <h4>Basketball</h4>
            <p>Maecenas bibendum lorem vitae turpis ullamcorper laoreet. Proin semper dictum risus in porta. In hac
              habitasse platea dictumst. Duis tincidunt, purus tortor fringilla orci, sit amet maximus urna velit eu
              urna.</p>
          </div>
          <div class="col-md-4 wow fadeInRight" data-wow-duration="2s">
            <div class="item-circle">
              <i class="fas fa-futbol fa-4x"></i>
            </div>
            <h4>Football</h4>
            <p>Maecenas bibendum lorem vitae turpis ullamcorper laoreet. Proin semper dictum risus in porta. In hac
              habitasse platea dictumst. Duis tincidunt, purus tortor fringilla orci, sit amet maximus urna velit eu
              urna.</p>
          </div>
        </div>
      </div>

    </div>
  </section>
  <section id="portfolio">
    <div class="container">
      <div class="portfolio-top text-center">
        <h3>OUR PORTFOLIO</h3>
        <p>Here are our portfolios</p>
      </div>
      <div class="portfolio-content text-center">
        <div class="row">
          <div class="col-md-4">
            <div class="card">
              <img src="images/product1.jpg" alt="Product1">
              <div class="card-block">
                <h4 class="card-title">Product1</h4>
                <p class="card-subtitle text-muted">Product1</p>
              </div>
            </div>
          </div>
          <div class="col-md-4">
            <div class="card">
              <img src="images/product2.jpg" alt="Product2">
              <div class="card-block">
                <h4 class="card-title">Product2</h4>
                <p class="card-subtitle text-muted">Product2</p>
              </div>
            </div>
          </div>
          <div class="col-md-4">
            <div class="card">
              <img src="images/product3.jpg" alt="Product3">
              <div class="card-block">
                <h4 class="card-title">Product3</h4>
                <p class="card-subtitle text-muted">Product3</p>
              </div>
            </div>
          </div>
        </div>
        <div class="row">
          <div class="col-md-4">
            <div class="card">
              <img src="images/product4.jpg" alt="Product4">
              <div class="card-block">
                <h4 class="card-title">Product4</h4>
                <p class="card-subtitle text-muted">Product4</p>
              </div>
            </div>
          </div>
          <div class="col-md-4">
            <div class="card">
              <img src="images/product5.jpg" alt="Product5">
              <div class="card-block">
                <h4 class="card-title">Product5</h4>
                <p class="card-subtitle text-muted">Product5</p>
              </div>
            </div>
          </div>
          <div class="col-md-4">
            <div class="card">
              <img src="images/product6.jpg" alt="Product6">
              <div class="card-block">
                <h4 class="card-title">Product6</h4>
                <p class="card-subtitle text-muted">Product6</p>
              </div>
            </div>
          </div>
        </div>
      </div>
  </section>
  <section id="aboutus">
    <div class="container">
      <div class="aboutus-top text-center">
        <h3>ABOUT US</h3>
        <p>Information about us</p>
      </div>
      <div class="aboutus-content">
        <div class="row">
          <div class="col-md-4 text-right">
            <h4>January 2017</h4>
            <h4>January</h4>
            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin non lectus id metus blandit efficitur
              vitae eu
              eros.
              Sed eget nunc tempor, convallis ligula non, vulputate ligula.</p>
          </div>
          <div class="col-md-4 text-center d-none d-sm-none d-md-block">
            <img src="images/january.jpg" class="image-circle ">
          </div>
        </div>
        <div class="row">
          <div class="col-md-4 offset-sm-4  text-center d-none d-sm-none d-md-block">
            <img src="images/april.jpg" class="image-circle">
          </div>
          <div class="col-md-4">
            <h4>April 2017</h4>
            <h4>April</h4>
            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin non lectus id metus blandit efficitur
              vitae eu
              eros.
              Sed eget nunc tempor, convallis ligula non, vulputate ligula.</p>
          </div>
        </div>
        <div class="row">
          <div class="col-md-4 text-right">
            <h4>July 2017</h4>
            <h4>July</h4>
            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin non lectus id metus blandit efficitur
              vitae eu
              eros.
              Sed eget nunc tempor, convallis ligula non, vulputate ligula.</p>
          </div>
          <div class="col-md-4 text-center d-none d-sm-none d-md-block">
            <img src="images/july.jpg" class="image-circle">
          </div>
        </div>
        <div class="row">
          <div class="col-md-4 offset-sm-4  text-center d-none d-sm-none d-md-block">
            <img src="images/september.jpg" class="image-circle">
          </div>
          <div class="col-md-4">
            <h4>September 2017</h4>
            <h4>September</h4>
            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin non lectus id metus blandit efficitur
              vitae eu
              eros.
              Sed eget nunc tempor, convallis ligula non, vulputate ligula.</p>
          </div>
        </div>
        <div class="row">
          <div class="col-md-4 text-right">
            <h4>November 2017</h4>
            <h4>November</h4>
            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin non lectus id metus blandit efficitur
              vitae eu
              eros.
              Sed eget nunc tempor, convallis ligula non, vulputate ligula.</p>
          </div>
          <div class="col-md-4 text-center d-none d-sm-none d-md-block">
            <img src="images/november.jpg" class="image-circle">
          </div>
        </div>
        <div class="row">
          <div class="col-md-4 offset-sm-4 d-none d-sm-none d-md-block">
            <div class="about-circle">
              <p class="yellow-text text-center">OUR <br>
                STORY<br>
                CONTINUES<br>
                ...<br></p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section id="team">
    <div class="container">
        <div class="team-top text-center">
          <h3>OUR AMAZING TEAM</h3>
          <p>Padfa xcsa hkgjdv hgsfa.</p>
        </div>
        <div class="team-content">
            <div class="row">
              <div class="col-md-4 text-center">
                <div class="team-circle"></div>
                <h4>John Black</h4>
                <h5>Swimming Trainer</h5>
                <ul class="social-icons">
                    <li class="social-icon">
                        <i class="fab fa-twitter"></i>
                        <li class="social-icon">
                          <i class="fab fa-facebook-f"></i>
                        </li>
                      </li>
                    <li class="social-icon">
                        <i class="fab fa-pinterest"></i>
                    </li>
                </ul>  
              </div>
              <div class="col-md-4 text-center">
                  <div class="team-circle"></div>
                  <h4>Jim Ukic</h4>
                  <h5>Basketball Trainer</h5>
                  <ul class="social-icons">
                      <li class="social-icon">
                          <i class="fab fa-twitter"></i>
                      </li>
                      <li class="social-icon">
                          <i class="fab fa-facebook-f"></i>
                      </li>
                      <li class="social-icon">
                          <i class="fab fa-pinterest"></i>
                      </li>
                  </ul>  
                </div>
                <div class="col-md-4 text-center">
                    <div class="team-circle"></div>
                    <h4>Mike Dutti</h4>
                    <h5>Football Trainer</h5>
                    <ul class="social-icons">
                        <li class="social-icon">
                            <i class="fab fa-twitter"></i>
                        </li>
                        <li class="social-icon">
                            <i class="fab fa-pinterest"></i>
                        </li>
                    </ul>  
                  </div>
              
            </div>
            <div class="row">
              <p>Proin condimentum, lorem a auctor lacinia, nisl mauris sollicitudin lectus, eget rutrum dolor augue sed mi. Pellentesque sed leo in tellus hendrerit tempor quis sit amet lorem. </p>
            </div>
        </div>
    </div>
  </section>
  <section id="partner">
    <div class="container">
      <div class="row text-center">
        <div class="col-md-3">
          <img src="images/partnerDataberry.jpg" alt="Car Wash" class="img-fluid">
        </div>
        <div class="col-md-3">
          <img src="images/partnerImmanuel.jpg" alt="Car Wash" class="img-fluid">
        </div>
        <div class="col-md-3">
          <img src="images/partnerMovers.jpg" alt="Car Wash" class="img-fluid">
        </div>
        <div class="col-md-3">
          <img src="images/partnerRocketGolf.jpg" alt="Car Wash" class="img-fluid">
        </div>
      </div>
    </div>
  </section>
  <section id="contact">
    <div class="container">
        <div class="contact-top text-center">
            <h3>CONTACT US</h3>
            <p>Praf hlaha jphsry hgayb.</p>
        </div>
        <div class="contact-content">
          <form role="form" method="post" class="golden-form">
            <div class="row">
              <div class="col-md-6">
                <div class="form-element">
                  <input type="text" class="golden-form-element" id="name" name="name" placeholder="YOUR NAME*">
                </div>
                <div class="form-element">
                  <input type="email" class="golden-form-element" id="email" name="email" placeholder="YOUR E-MAIL*">
                </div>
                <div class="form-element">
                  <input type="text" class="golden-form-element" id="subject" name="subject" placeholder="SUBJECT">
                </div>
              </div>
              <div class="col-md-6">
                <div class="form-element">
                  <textarea name="message" id="message" class="golden-form-element" placeholder="YOUR MESSAGE*"></textarea>
                </div>
              </div>
            </div>
            <div class="row justify-content-center">
              <div class="form-element">
                <input type="submit" class="btn btn-primary btn-lg" id="submit" name="submit" value="SEND MESSAGE">
              </div>
            </div>
          </form>
        </div>
    </div>
  </section>
  <section id="footer">
    <div class="container">
      <div class="row">
        <div class="col-md-4">
          <p>&copy; Copyright 2019 ugunay</p>
        </div>
        <div class="col-md-4">
          <ul class="company-social-icons text-center">
            <li class="company-social-icon">
              <i class="fab fa-twitter"></i>
            </li>
            <li class="company-social-icon">
              <i class="fab fa-facebook-f"></i>
            </li>
            <li class="company-social-icon">
              <i class="fab fa-pinterest"></i>
            </li>
            <li class="company-social-icon">
              <i class="fab fa-google-plus"></i>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </section>
  <!-- Optional JavaScript -->
  <!-- jQuery first, then Popper.js, then Bootstrap JS -->
  <script src="js/jquery-3-3-1.min.js"></script>
  <script src="js/bootstrap.bundle.min.js"></script>
  <script src="js/wow.min.js"></script>
  <script>
    new WOW().init();
  </script>
  <script src="js/main.js"></script>
</body>

</html>